README.md

```diff
@@ -249,53 +277,142 @@ pre-commit install
 ```
 
 
-## Running DataGateway API
+
+
+# Running DataGateway API
 Depending on the backend you want to use (either `db` or `python_icat`) the connection
```

Comment:
Could have "see more details [here]" style link to the section on the backend here?

Reply:
Added an link to a section lower down in the README.